web/src/components/netflow-traffic.tsx

```diff
@@ -219,22 +195,14 @@ export const NetflowTraffic: React.FC<NetflowTrafficProps> = ({
   }, [getAvailableColumns]);
 
   const getFilterDefs = React.useCallback(() => {
-    return getFilterDefinitions(model.config.filters, model.config.columns, t).filter(
-      fd =>
-        (isMultiCluster() || fd.id !== 'cluster_name') &&
-        (isZones() || !fd.id.endsWith('_zone')) &&
-        (isConnectionTracking() || fd.id !== 'id') &&
-        (isDNSTracking() || !fd.id.startsWith('dns_')) &&
-        (isPktDrop() || !fd.id.startsWith('pkt_drop_')) &&
-        (isFlowRTT() || fd.id !== 'time_flow_rtt') &&
-        (isUdn() || fd.id !== 'udns') &&
-        (isPktXlat() || !fd.id.startsWith('xlat_')) &&
-        (isNetEvents() || fd.id !== 'network_events') &&
-        (!isPromOnly() || checkFilterAvailable(fd, model.config.promLabels)) &&
-        (isIPSec() || !fd.id.startsWith('ipsec_'))
-    );
+    return getFilterDefinitions(model.config.filters, model.config.columns, t).filter(fd => {
+      if (fd.id === 'id') {
+        return isConnectionTracking();
+      }
+      return checkFilterAvailable(fd, model.config, model.dataSource);
+    });
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [model.config.columns, model.config.filters, model.config.promLabels, isPromOnly]);
+  }, [model.config, model.dataSource]);
```

Comment:
I think there was some exceptions here:
- You already managed the UDN one
- I see drop state and cause filters when packet drop feature is not enabled

Reply:
Others filters seems to work properly :ok_hand:

Reply:
did you take the operator PR too? The issue with drops is supposed to be fixed from the operator config

Reply:
I was using it yes

Reply:
weird, I don't have them
![image](https://github.com/user-attachments/assets/4a88ab08-b696-4085-bc66-d795b6a3f012)

Reply:
the operator generated image was prior to this commit, I guess you used that old one; just triggered a new one